envs/env-py311.yml

```diff
@@ -147,6 +149,7 @@ dependencies:
   - pypdf2
   - pyqt >=5.15.0
   - pyqtgraph
+  # - pyside6
```

Comment:
Same comment here.

Reply:
The files are identical besides the Python version, therefore there are the same commented lines.

Reply:
I've posted diffs here: https://github.com/NSLS2/nsls2-collection-tiled/pull/64#issuecomment-2867624623.